pkg/river/internal/value/value_test.go

```diff
@@ -64,6 +100,10 @@ func (*pointerMarshaler) MarshalText() ([]byte, error) {
 	return []byte("Hello, world!"), nil
 }
 
+type Body struct {
+	Data pointerMarshaler `river:"data,attr"`
+}
```

Comment:
@erikbaranowski Since this is being declared for the scope of the one test, and it doesn't have any methods, can we move it to the top of the inside of the function?

```go
func TestInterfacePointerReceiver() {
  type Body { ... } 

  ...
}